internal/pkg/instrumentation/bpf/net/http/server/bpf/probe.bpf.c

```diff
@@ -26,13 +26,19 @@ char __license[] SEC("license") = "Dual MIT/GPL";
 #define MAX_CONCURRENT 50
 #define W3C_KEY_LENGTH 11
 #define W3C_VAL_LENGTH 55
+#define REMOTE_ADDR_MAX_LEN 32
+#define HOST_MAX_LEN 32
```

Comment:
I think this needs to be 253 to support https://www.rfc-editor.org/rfc/rfc7230.html#section-5.4 right?

Or maybe 255 given this will include the port, right?

Reply:
Yeah, host will be the full string value so needs to be 255.

Reply:
change both REMOTE_ADDR_MAX_LEN and HOST to 256